### PR TITLE
Do not include scribe sink when compiling on non x86 platforms.

### PR DESCRIPTION
**Pull request by kramasamy**
This is due to scribe dependency on thrift which needs to generate java files - the binaries used are only x86 platforms.

### CODE REVIEW

File: tools/platform/BUILD [hunk 1]
```diff
@@ -7,15 +7,7 @@ config_setting(
...
-    name = "ubuntu",
-    values = {
-        "cpu": "k8",
-    },
-    visibility = ["//visibility:public"],
-)
-
-config_setting(
-    name = "centos",
+    name = "k8",
```
Comment (nwangtw) [inline tools/platform/BUILD]: I am wondering if the change in this file is expected/needed in this PR. it seems unrelated and looks a little strange.

Comment (kramasamy) [reply tools/platform/BUILD]: good catch it is not needed.

Comment (kramasamy) [reply tools/platform/BUILD]: @nwangtw - i need to take my comment back. this is needed to ensure all linux variants are grouped under one target which is k8 - otherwise, it is ubuntu, centos and more variations.

Comment (nwangtw) [reply tools/platform/BUILD]: I see. So now k8 is considered as a platform? A little unclear but not a blocker to me.

**Comment by kramasamy** [comment]
@nwangtw - k8 represents all variants of linux in bazel.
